docker/Dockerfile

```diff
@@ -68,8 +63,6 @@ ENV VCPKG_DEFAULT_BINARY_CACHE=/root/vcpkg_cache
 WORKDIR /
 COPY vcpkg.json .
 RUN <<EOF
-  set -ex
-
   # Install the manifested packages.
```

Comment:
We should introduce a list of all supported compilers and install the packages in a loop.